docgenhtml/plugins/org.polarsys.capella.docgen/egf/HTMLDocGenCapella.fcore

```diff
@@ -65,7 +65,7 @@
             <methods xmi:id="_TM8hYPbpEd-RH7MH9LLQsg" name="setFileNameService" patternFilePath="templates/pattern._ixj0MN0eEd-Vzvs7Wt1vzQ/method._TM8hYPbpEd-RH7MH9LLQsg.pt"/>
             <methods xmi:id="_5jqVAErlEeCvqtVx_IKrqA" name="endContent" patternFilePath="templates/pattern._ixj0MN0eEd-Vzvs7Wt1vzQ/method._5jqVAErlEeCvqtVx_IKrqA.pt"/>
             <methods xmi:id="_ALikYLklEemEHKw9GhkeOQ" name="docHeader" patternFilePath="templates/pattern._ixj0MN0eEd-Vzvs7Wt1vzQ/method._ALikYLklEemEHKw9GhkeOQ.pt"/>
-            <variables xmi:id="_8ZnqQN4gEd-Qav1YnqlO9A" name="element" type="http://www.polarsys.org/capella/core/core/5.0.0#//CapellaElement"/>
+            <variables xmi:id="_8ZnqQN4gEd-Qav1YnqlO9A" name="element" type="org.polarsys.capella.core.data.capellacore.CapellaElement"/>
```

Comment:
Please use nsuri based classes URI in all patterns. You have juste to replace the version number.

Reply:
The change https://github.com/eclipse/kitalpha/commit/2baea9197d05d6c8e8f0f9d3ebb3040ef9eddc3f is meant to allow to use qualified names instead of URI. It will prevent future refactoring when version number will change in the Capella M2